spec/coverage_statistics_spec.rb

```diff
@@ -3,7 +3,7 @@
 require "helper"
 
 RSpec.describe SimpleCov::CoverageStatistics do
-  describe ".new" do
+  describe "::new" do
```

Comment:
? That's not the convention I usually follow in Ruby. For me it's `.class_method` and `#instance_method` :)

Reply:
The `::` convention is used [in Ruby Docs](https://ruby-doc.org/core-2.2.0/BasicObject.html) for public class methods (see the list of methods on the left), but I do think the RSpec convention is to use `.`. I'll change it back. Thanks for reviewing!